.circleci/config.yml

```diff
@@ -28,31 +24,38 @@ references:
       path: junit.xml
       prefix: tests
 
+# define images as executor/not via 'reference' so that renovate will pick it 
+# up for updates
+executors:
+  node:
+    resource_class: medium+
+    docker:
+      # On Circle-CI We have to use a build images approved by SAP (for build compliance)
+      # see https://wiki.one.int.sap/wiki/pages/viewpage.action?pageId=2650697023
+      - image: node:20
```

Comment:
@TorgeHarbig Could we use 22? The old config uses lts which should be 22 for now.

Reply:
Yes we can also use node:22. I do the update and then let's check the pipeline again.